libs/components/ag-grid/src/lib/modules/ag-grid/cell-renderers/cell-renderer-row-selector/cell-renderer-row-selector.component.spec.ts

```diff
@@ -174,11 +174,40 @@ describe('SkyCellRendererCheckboxComponent', () => {
         rowSelectorCellComponent.rowNode?.setSelected
       ).toHaveBeenCalledWith(true);
     });
+
+    it(`should not set the rowNode selected property to the component's checked property value if no column field provided and value is not changed`, () => {
+      const rowNode = new RowNode({} as Beans);
+      rowNode.data = {};
+      cellRendererParams.node = rowNode;
+      cellRendererParams.colDef = {
+        field: undefined,
+      };
+      spyOn(rowNode, 'isSelected').and.returnValues(true, true);
+
+      rowSelectorCellFixture.detectChanges();
+      rowSelectorCellComponent.agInit(
+        cellRendererParams as ICellRendererParams
+      );
+
+      spyOn(rowNode, 'setSelected');
+
+      rowSelectorCellComponent.updateRow();
+
+      expect(
+        rowSelectorCellComponent.rowNode?.setSelected
+      ).not.toHaveBeenCalled();
+    });
   });
 
   describe('refresh', () => {
     it('returns false', () => {
-      expect(rowSelectorCellComponent.refresh()).toBe(false);
+      expect(
```

Comment:
I think this `it` description needs to be updated

Reply:
Good catch. Updated.